File: 1. Tests/LiveDocumentor.Test.VBDocumentation/ExtensionMethods.vb
Imports System.Runtime.CompilerServices

Public Class ExtensionMethods
    Public Shared Function WhereEntityIs(
        self As IEnumerable(Of String),
        metadata As String()
        ) As IEnumerable(Of String)
        WhereEntityIs = New List(Of String)()
    End Function
End Class

File: Source/1. Tests/Framework.Tests/v4.0/v4/ExtensionMethods.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace v4._0.v4
{
    public static class ExtensionMethods
    {
        /// <summary>
        /// An extension method in .net using dynamics
        /// </summary>
        /// <param name="self"></param>
        /// <param name="metadata"></param>
        /// <returns></returns>
        public static IEnumerable<string> WhereEntityIs(this IEnumerable<string> self, params string[] metadata)
        {
            return new List<string>();
        }
    }
}
